tcms/testruns/static/testruns/js/get.js

```diff
@@ -207,8 +207,8 @@ function renderTestExecutions(testExecutions) {
     $('.test-executions-count').html(testExecutions.length)
 }
 
-function bindEvents() {
-    treeViewBind()
+function bindEvents(selector) {
```

Comment:
Does bindEvents() on 205 need a parameter after this change or not ?

Reply:
no, because by default it's undefined and then `treeViewBind` has other default value which will be used